src/index.ts

```diff
@@ -35,13 +51,20 @@ export function init({
     wallet.create(
       new Web3AuthConnection(network, clientId, mfaLevel, uxMode),
       new FlowportApiConnection(web3AuthNetworkToFlowportApiMapping[network]),
+      getDefaultWalletCallbacks(),
     ),
     web3AuthFclServices,
   )
   listenToMessages(api)
 }
 
+export function config(callbacks: Partial<WalletActionsCallbacks>) {
```

Comment:
I'd consider merging this `config()` into the `init` method, I don't see a usecase to do the initialization and callbacks configuration separately, that way all the setup of the wallet is kept in a single method which makes the API easier to understand/use

Reply:
I see a usecase for the config being a separate function. `init` should be called as soon as possible, lets say in the `App` components `useEffect`, whereas callbacks for tx confirm or user login might be available somewhere deep down the component tree. I updated the function so it updates just the passed callbacks, not all of them, so dapp can call `config({onConfirmSign})` in one place and `config({onLoadingStateChange})` somewhere else.

Reply:
makes sense, let's have it separate then. Though I'd consider at least renaming `config()` to `setCallbacks()` as the name `config()` may lead the reader to believe tha the call allows configuring also other stuff like api keys, flags, ... which isn't the case